### PR TITLE
[observability pipelines worker] 1.1.1 release

### PR DESCRIPTION
**Pull request by spencergilbert**
#### What this PR does / why we need it:

~~The original PR intending to GA the chart and application missed the `image.tag` value, this wasn't a _huge_ deal as the `1.0.0` was a retag of the old `0.1.0` so by happy accident nothing was broken for users.
I've updated the release documentation to include this change as well, but also considering removing the number of places this needs to be updated to avoid future mistakes.~~

Repurposing for the upcoming release, given the we're moving past the `1.0` quickly. I also removed the extra places that needed version bumps to streamline releases.

#### Which issue this PR fixes

  - closes [OPB-806](https://datadoghq.atlassian.net/browse/OPB-608)
 
#### Special notes for your reviewer:

~~The image was incorrectly published as `v1.0.0`, this PR can't merge until the correct image has been released.~~

Awaiting the upcoming `1.1.1` release for OPW.

#### Checklist

- [x] Documentation has been updated with helm-docs (run: `.github/helm-docs.sh`)
- [x] Chart Version bumped
- [x] `CHANGELOG.md` has been updated
- [x] Variables are documented in the `README.md`

### CODE REVIEW

**Comment by 001wwang** [comment]
Are any modifications required in relation to: https://github.com/DataDog/observability-pipelines-worker/pull/27?

TL;DR: The binary is run as `observability-pipelines-worker run {file}` instead of `observability-pipelines-worker --config ...`

**Comment by spencergilbert** [comment]
> Are any modifications required in relation to: [DataDog/observability-pipelines-worker#27](https://github.com/DataDog/observability-pipelines-worker/pull/27)?
> 
> TL;DR: The binary is run as `observability-pipelines-worker run {file}` instead of `observability-pipelines-worker --config ...`

👍 thanks for the callout, will make a commit to update the args to `run /etc/opw/config.yaml`

As I looked at that linked PR, I noticed the `DATA_DIR` being `/var/lib/datadog-observability-pipelines-worker/`

Right now the chart has this for it's `data` volume:

```yaml
    volumeMounts:
      - name: data
        mountPath: "{{ .Values.config.data_dir | default "/var/lib/opw" }}"
```

Should that default be updated to match?

**Comment by 001wwang** [comment]
> As I looked at that linked PR, I noticed the `DATA_DIR` being `/var/lib/datadog-observability-pipelines-worker/`
> 
> Right now the chart has this for it's `data` volume:
> 
> ```yaml
>     volumeMounts:
>       - name: data
>         mountPath: "{{ .Values.config.data_dir | default "/var/lib/opw" }}"
> ```
> 
> Should that default be updated to match?

Yes. Per https://datadoghq.atlassian.net/browse/OPB-566, `/var/lib/datadog-observability-pipelines-worker/` is what we're moving forward with.

**Review by 001wwang — APPROVED** [review]

**Comment by 001wwang** [comment]
The image for 1.1.0 has been published: https://gitlab.ddbuild.io/DataDog/public-images/-/jobs/229854271
- Overall pipeline: https://gitlab.ddbuild.io/DataDog/observability-pipelines-worker/-/pipelines/13373173

**Comment by spencergilbert** [comment]
@001wwang updated the image and the chart to 1.1.1 per my last instructions

**Review by 001wwang — APPROVED** [review]
Confirming that, due to issues with `1.1.0`, we are releasing OPW `1.1.1` (https://github.com/DataDog/observability-pipelines-worker/releases/tag/1.1.1). The `1.1.1` image has been published as of https://gitlab.ddbuild.io/DataDog/public-images/-/pipelines/13510108.
